src/components/dropdown-menu/DropdownMenu.tsx

```diff
@@ -14,18 +14,44 @@ interface DropdownMenuItemProps extends React.HTMLProps<HTMLDivElement> {
   children?: React.ReactNode;
   className?: string;
   onClick?: (event: React.MouseEvent<HTMLElement>) => void;
+  // To select a certain option in the menu without having to use document.querySelector
+  customRef?: React.RefObject<HTMLDivElement>;
```

Comment:
Minor detail, in the icon component we name it `forwardRef`, I think that you have the same use here, do you mind aligning the name please?
https://github.com/SymphonyPlatformSolutions/symphony-bdk-ui-toolkit-components/blob/master/src/components/icon/Icon.tsx#L12